server/log/main.go

```diff
@@ -133,6 +157,16 @@ func main() {
 		glog.Fatalf("Failed to load server key: %v", err)
 	}
 
+	// Start HTTP server (optional)
+	if *exportRpcMetrics {
```

Comment:
In theory they can be shared. In Go 1.7 expvar registers on the default ServeMux so I suspect it might not work properly until Go 1.8. Haven't proven this yet though.